LemonadeGui.py

```diff
@@ -135,8 +135,11 @@ def draw_help(self):
         if self.game_mode == 0:
             block = self._blit_to_block([
                 _("""Type in the number you want to buy of each item"""),
-                _("""(be careful: you will need all 3 """
-                  """supplies to make the lemonade)""")])
+                _(
+                    """
```

Comment:
Wrong indentation.

Reply:
This causes Flake8(E122)
```
                _(
                """
                (be careful: you will need all 3 supplies to make the lemonade)
                """
                )])
````

Reply:
Then fix it.

Reply:
The following causes Flake8(E122)
```
_(
"""
(be careful: you will need all 3 supplies to make the lemonade)
"""
)])
```

That's why I intended it to look like this so it doesn't cause a Flake8(E122)
```
 _(
    """
(be careful: you will need all 3 supplies to make the lemonade)
"""
)])
```

Do you suggest a different approach?

Reply:
This seems to fix it;
```python3
         if self.game_mode == 0:
             block = self._blit_to_block([
                 _("""Type in the number you want to buy of each item"""),
                 _("""be careful: you will need all 3 supplies to make \
                 the lemonade""")
                 ])
```

Reply:
Fixed